usaspending_api/references/v2/views/toptier_agencies.py

```diff
@@ -1,15 +1,26 @@
 from django.db.models import F, Sum
 from django.db.models.functions import Coalesce
-from usaspending_api.references.models import Agency
+
+from usaspending_api.common.helpers.date_helper import now
+from usaspending_api.references.models import Agency, GTASSF133Balances
 from usaspending_api.common.cache_decorator import cache_response
+from usaspending_api.references.v2.views.agency import get_total_budgetary_resources
 from usaspending_api.submissions.models import SubmissionAttributes
 
 from rest_framework.response import Response
 from rest_framework.views import APIView
 from usaspending_api.common.exceptions import InvalidParameterException
 from usaspending_api.accounts.models import AppropriationAccountBalances
 
-from usaspending_api.references.constants import TOTAL_BUDGET_AUTHORITY
+
+def get_total_obligations_incurred(fiscal_year, fiscal_period):
```

Comment:
Seems like a copy paste of the above function. Is there a good central place for this to live similar to that constant this is replacing?

Reply:
The functions are slightly different - this one is pulling `obligations_incurred_total_cpe` not `total_budget_authority_amount_cpe`. We could move both functions to thee `constants.py` file where the old constants were, but since they're functions and not constants and are really only used in these two files I just left them where they're used. I'm fine with either tactic.

Reply:
Ah I didn't see that difference. Works for me then.